h_program-lang/AST_generic.ml

```diff
@@ -592,7 +592,7 @@ and stmt =
   (* EmptyStmt = Block [], or separate so can not be matched by $S? $ *)
 
   (* newscope: for vardef in expr in C++/Go/... *)
-  | If of tok (* 'if' or 'elif' *) * expr * stmt * stmt
+  | If of tok (* 'if' or 'elif' *) * expr * stmt * stmt option
```

Comment:
I was wondering if we wanted to treat:

```java
if (cond) { ... } else {}
```

and

```java
if (cond) { ... }
```

as the same code. If we do, it might make sense to just keep the else block as an empty list? 🤷 

Note that in the motivating issue, they want to ensure that the else block uses an external variable.